lib/locale_threads.t

```diff
@@ -54,67 +470,1290 @@ SKIP: { # perl #127708
     pass("Didn't segfault");
 }
 
+# Second test setup
+my %locale_name_to_object;
+for (my $i = 0; $i < @locales; $i++) {
```

Comment:
Is there a reason this is a C style loop instead of:
```
for my $i (0 .. $#locales) { # or @locales-1
```
There's other cases further down.

Reply:
Just habit from mostly programming in C these days.  Changed.